schrodinger_build.py

```diff
@@ -0,0 +1,82 @@
+"""
```

Comment:
Meh, let's leave this out until it's something that works.

Reply:
Deleted.